docs/development/schema/owasp.threat-dragon.schema.json

```diff
@@ -0,0 +1,407 @@
+                                "type": "string"
+                              },
+                              "status": {
+                                "description": "The threat status as NA, Open or Mitigated",
+                                "type": "string"
+                              },
+                              "threatId": {
+                                "description": "The threat ID as a UUID",
+                                "type": "string",
+                                "minLength": 36
+                              },
+                              "title": {
+                                "description": "The threat title",
+                                "type": "string"
+                              },
+                              "type": {
+                                "description": "The threat type, selection according to modelType",
+                                "type": "string"
+                              }
+                            },
+                            "required": [ "description", "mitigation", "severity", "status", "title", "type" ]
+                          }
+                        },   
+                        "type": {
+                          "description": "The component type",
+                          "type": "string"
+                        },
+                        "vertices": {
+                          "description": "The boundary or data-flow curve points",
+                          "type": "array",
+                          "items": {
+                            "type": "object",
+                            "properties": {
+                              "x": {
+                                "description": "The horizontal value of the curve point",
+                                "type": "integer"
+                              },
+                              "y": {
+                                "description": "The vertical value of the curve point",
+                                "type": "integer"
+                              }
+                            },
+                            "required": [ "x", "y" ]
+                          }
+                        },   
+                        "z": {
+                          "description": "The component Z-plane",
+                          "type": "integer"
+                        }
+                      },
+                      "required": [ "attrs", "id", "size", "type", "z" ]
+                    }
+                  }
+                }
+              }
+            },
+            "required": [ "diagramType", "id", "size", "thumbnail", "title", "diagramJson" ]
+          }
+        },
+        "reviewer": {
+          "description": "The reviewer of the overall threat model",
+          "type": "string"
+        }
+      },
+      "required": [ "contributors", "diagrams" ]
+    }
+  },
+  "required": [ "summary", "detail" ]
+}
```

Comment:
Will validate today

Reply:
Note that this is the version 1.x schema, and we are hoping that version 2.0 Threat Dragon (which is still in development) will keep to this schema
At present a version 2.0 threat model will fail the schema - but we are working on that

Reply:
and I have corrected an error where the 'tdVersion' was in the wrong place